sdcm/utils/common.py

```diff
@@ -2756,6 +2756,54 @@ def clean_enospc_on_node(target_node, sleep_time):
     target_node.wait_db_up()
 
 
+def verify_quota_is_configured_for_scylla_user(target_node) -> bool:
```

Comment:
put it to separate file rather than `common.py` - it is not that common (maybe smthing like `xfs_quota.py`?)

Reply:
I can put it to separate file but why we don't create a separate file for enospc methods then?